proto/cosmos/auth/v1beta1/auth.proto

```diff
@@ -36,6 +36,15 @@ message ModuleAccount {
   repeated string permissions  = 3;
 }
 
+// ModuleCredential represents a unclaimable pubkey for base accounts
+//
+// Since: cosmos-sdk 0.47
+message ModuleCredential {
+  string         module_name        = 1;
+  bytes          module_account_key = 2;
```

Comment:
```suggestion
  // module_account_key is the optional key used for a module sub-account.
  bytes          module_account_key = 2;
```

Reply:
I don't agree. Module subaccounts are not derived: https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-028-public-key-addresses.md#module-account-addresses
```
Hash("module", "group\0subacct") // What group currently has
```
Which is not equal to
```
Derive(Hash("module", "group"), "\0subacct")
```

Reply:
Updated to support only one way, as there is discussion to simplify ADR-38.

Reply:
Created https://github.com/cosmos/cosmos-sdk/issues/13910 for discussing this.

Reply:
The motivation for `subModuleAccount = Hash("module", append(mKey, key...))` was to use `"module"` as a root type, rather than `"module\0<module_name"`. I will describe it more in #13910